watson/cli.py

```diff
@@ -414,8 +419,11 @@ def report(watson, from_, to, projects, tags):
               help="Logs activity only for frames containing the given "
               "tag. You can add several tags by using this option multiple "
               "times")
+@click.option('-m', '--matcher', 'matcher',
```

Comment:
The `-m` option is now already used as the short form of `--month` and conflicts with the usage here.